src/screens/DataDisplayScreen.tsx

```diff
@@ -0,0 +1,43 @@
+
+import React, { useEffect, useState } from 'react';
+import { View, Text, FlatList } from 'react-native';
+import { User } from '../types/Types';
+import { getDataDisplay } from '../apis/DataDisplayApi';
+
+const DataDisplayScreen: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    try {
+      const response = await getDataDisplay();
+      setUsers(response.users);
```

Comment:
This is the sample payload for `DataDisplayResponse`:

{"userDetailsList":[{"userId":"vap1","firstName":"vaibhav","lastName":"vasant","email":"v@gmail.com","phoneNumber":"7633824597","address":"None"},{"userId":"vap2","firstName":"vaibhav","lastName":"vasant","email":"v@gmail.com","phoneNumber":"7633824597","address":"None"}]}

Based on that change `response.users` to `response.userDetailsList`